src/scripts/main.js

```diff
@@ -1,3 +1,62 @@
 'use strict';
 
 // write code here
+
+function toNumber(Str) {
+  const numb = +Str.replace(/\D/g, '');
+
+  return numb;
+}
+
+const sortAll = (e) => {
+  const arraySort = [...listElements].sort((a, b) => {
+    if (e.target === document.querySelectorAll('th')[0]) {
```

Comment:
It is better to use a switch...case if you have a limited amount of possible values
https://www.w3schools.com/js/js_switch.asp